sm/webapp/static/js/chart.js

```diff
@@ -1,112 +1,58 @@
+function centroidPlot(data) {
+    var mzs = data['mzs'];
+    var ints = data['ints'];
+    var mzs2 = [];
+    var ints2 = [];
+    for (var i = 0; i < mzs.length; i++) {
+        mzs2.push(mzs[i] - 1e-6);
+        mzs2.push(mzs[i]);
+        mzs2.push(mzs[i] + 1e-6);
```

Comment:
It's a horrible hack to have (almost) vertical lines as centroids. I tried to use bar plots first, but there are no options to specify bar width yet, so I resorted to a line plot.